index.html

```diff
@@ -1,13 +1,61 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="style.css">
     <title>Document</title>
 </head>
+
 <body>
-   <h1 class="color-blue">Hello Microverse</h1> 
+    <header>
+        <div class="head">
+            <div class="logo">
+                <p>My logo</p>
+            </div>
```

Comment:
- Kindly use a header container here. This is because your Logo is in the introductory part of your page hence why it is important for you to use a proper Html semantics. You can read up more [here](https://github.com/microverseinc/curriculum-html-css/blob/main/html5.md)